javascript/javascript2/week1/Week1HW.js

```diff
@@ -0,0 +1,89 @@
+"use strict";
+
+//Find the shortest word
+
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+const shortestWord = getShortestWord(danishWords); // returns 'ø'
+function getShortestWord(words) {
+  let newWord = danishWords[0];
+  for (let i = 0; i < danishWords.length; i++) {
```

Comment:
In the first loop iteration your code will always compare the first word with itself!
E.g., here in the first iteration both word and shortestWord are "bil"
Which means there is always one unnecessary iteration. Can you change the for loop such that this is solved?

Reply:
yes , I changed i = 1 so that it starts from the second index.